code/read_raw.R

```diff
@@ -13,12 +9,37 @@ read_parlamentares_raw <-
       col_types = cols(
         .default = col_character(),
         em_exercicio = col_double(),
-        is_parlamentar = col_double()
+        is_parlamentar = col_double(), 
+        legislatura = col_double()
       )
     ) %>%
-      select(-legislatura) %>%
-      distinct()
+      filter(is_parlamentar == 1, em_exercicio == 1) %>% 
```

Comment:
Esse filtro atual faz com que parlamentares fora da legislatura 56 sejam capturados. Acho que faz sentido adicionar outro filtro legislatura == 56 para garantir que apenas parlamentares dessa legislatura sejam filtrados. Ao adicionar esse filtro 593 parlamentares são retornados (1 a menos do que o esperado). 

Obs: Sobre a geração do entidades.csv, vamos investigar a falta desse/a parlamentar e também o motivo pelo qual a coluna em_exercicio está retornando parlamentares de outras legislaturas também.
```suggestion
      filter(is_parlamentar == 1, em_exercicio == 1, legislatura == 56) %>% 
```